### ISSUE TITLE
Parent selector interpolation not working?

### ISSUE THREAD

**Issue by zlobec** [leafo/scssphp#614]
I have the v0.7.6 and the parent selector interpolation is not working when defined as:
```
display {
   $prefix: &;
   @at-root {
     [class^="#{$prefix}"] {
     text-transform: uppercase;
   }
 }
}
```
I get the error: parse error: failed at `$c: &amp;;` (stdin)

I managed to get the workaround, but I need to define the parent as a string variable.
```
display {
   @at-root {
     $prefix: display;
     [class^="#{$prefix}"] {
       text-transform: uppercase;
     }
   }
}
```
Or I'm i doing something wrong? Thanks for the response.

**Comment by James90** [comment]
I have a similar problem with v0.7.7 and  twbs/bootstrap v4.2.1. 

Error: ```parse error: failed at `&} {` ``` in mixins/_badge.scss on line 5:

```scss
@mixin badge-variant($bg) {
  color: color-yiq($bg);
  background-color: $bg;

  @at-root a#{&} {
    @include hover-focus {
      color: color-yiq($bg);
      background-color: darken($bg, 10%);
    }
  }
}
```

**Comment by Cerdic** [comment]
This is fixed in last version with #643

**Comment by Cerdic** [comment]
(can be closed @robocoder)
